test/TensorFlowRuntime/top_level_2.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+
+// This file contains stubs for functions that the playground transform in
+// lib/Sema/PlaygroundTransform.cpp generates calls into.
+
+import TensorFlow
+import TensorFlowUnittest
+import StdlibUnittest
+
+var TopLevelTests = TestSuite("TopLevel")
+
+let g = Tensor<Float>(1.0)
+
+// This function is not inlined into main(). Confirm that it 
```

Comment:
Stale comment here and below?

Anyhow, is the test supposed to make sure that there are no sends or receives?

Reply:
Thanks for the catch! Fixed.

This is to check for correctness. There will be sends/recvs, but I believe warning is disabled for the main() function (search for `if (hostFn.getName() == SWIFT_ENTRY_POINT_FUNCTION)` in TFPartition.cpp).